### PR TITLE
lalezar-fonts: init at unstable-2017-02-28

### PR DESCRIPTION
**Pull request by LinArcX**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by LinArcX** [comment]
@GrahamcOfBorg build lalezar-fonts

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6552,9 +6555,6 @@ in
   yaft = callPackage ../applications/misc/yaft { };
 
   yarn = callPackage ../development/tools/yarn  { };
-  
-  yarn2nix = throw "Use upstream https://github.com/moretea/yarn2nix";
-  mkYarnPackage = yarn2nix;
```
Comment (veprbl) [inline pkgs/top-level/all-packages.nix]: Also here

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -10,6 +10,7 @@ res: pkgs: super:
 
 with pkgs;
 
+
```
Comment (veprbl) [inline pkgs/top-level/all-packages.nix]: And here

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (LinArcX) [reply pkgs/top-level/all-packages.nix]: @veprbl Done!

**Comment by LinArcX** [comment]
why closed?

**Comment by veprbl** [comment]
@LinArcX You did not fix the spurious change, so I did it for you. This is in master now.

**Comment by LinArcX** [comment]
@veprbl Thank you.

**Pull request closed without merging**